### ISSUE TITLE
Remote Config Parsing Issue

### ISSUE THREAD

**Issue by SupImDos** [SeydX/homebridge-bravia-tvos#127]
Relevant Code:
https://github.com/SeydX/homebridge-bravia-tvos/blob/258fae2e4797ee09a994f5d82b2031dfa24b6dc8/src/platform.js#L125

Example Relevant Config from ConfigUIX:
```
"remote": [
    {
        "command": "Home",
        "target": "INFORMATION"
    }
```

Not sure if I am missing something here.

When I configure a custom remote button using the Config UI, it generates the above config entry.

In the code, it looks like the `cmd.command` attribute is tested against the regex for the actual `ircc` command instead of the `name` of the command.

Not sure if this is something that should be fixed here, or in the Config UI generation.

**Comment by simonrb2000** [comment]
This may explain why mine do not work, I’ve been using the Config UI generator.

**Comment by SeydX** [comment]
@SupImDos 

Thx, will fix this in the next version

**Comment by SeydX** [comment]
should be fixed with v4.1.10

**Comment by SupImDos** [comment]
> should be fixed with v4.1.10

Thank you! I am not home for a few days, but I will test this as soon as I can and close the issue.

**Comment by Austin519** [comment]
This does not yet seem to be fixed.  I'm also having trouble binding Apple Remote buttons to new commands.

**Comment by SeydX** [comment]
Hey sry for that, should be fixed now with v4.1.12
